### PR TITLE
[breakpad] Support breakpad_client on Linux & macOS

### PR DESCRIPTION
**Pull request by Perlmint**
Support breakpad_client on Linux & macOS

Tested on macOS Big Sur and Ubuntu 20.04 docker on macOS.

### CODE REVIEW

File: ports/breakpad/portfile.cmake [hunk 1]
```diff
@@ -3,14 +3,29 @@ vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
...
     OUT_SOURCE_PATH SOURCE_PATH
     REPO google/breakpad
-    REF 9c4671f2e3a63c0f155d9b2511192d0b5fa7f760 # accessed on 2020-09-14
+    REF 9c4671f2e3a63c0f155d9b2511192d0b5fa7f760 # accessed on 2020-09-14
     SHA512 4c9ed9b675a772f9a6a84692865381130901820cb395b725511e7a9e2cbf4aaa5212a9ef5f87086baf58bb9d729082232b564bd827a205f87b5c1ffc1c53892a
     HEAD_REF master
     PATCHES
         fix-unique_ptr.patch
 )
+if(VCPKG_CMAKE_SYSTEM_NAME STREQUAL Linux)
```
Comment (JackBoosY) [inline ports/breakpad/portfile.cmake:12]: Is this for Linux and OSX?

**Review by JackBoosY — CHANGES_REQUESTED** [review]

File: ports/breakpad/portfile.cmake (hunk 1 above)
Comment (Perlmint) [reply ports/breakpad/portfile.cmake:12]: This part is required for only Linux, not OSX.

Comment (JackBoosY) [inline ports/breakpad/portfile.cmake:12]:
```suggestion
if(VCPKG_HOST_IS_LINUX)
```

**Review by JackBoosY — CHANGES_REQUESTED** [review]

**Review by JackBoosY — APPROVED** [review]

**Comment by JackBoosY** [comment]
What's going on with the license check?

**Comment by JackBoosY** [comment]
@Perlmint Can you re-sign the CLA please?

**Comment by Perlmint** [comment]
> @Perlmint Can you re-sign the CLA please?

I just opened the https://cla.opensource.microsoft.com/microsoft/vcpkg?pullRequest=15092. But, there is no submit/sign button. How can I re-sign the CLA?

**Comment by JackBoosY** [comment]
Hmm... @BillyONeal can you please take a look?

File: ports/breakpad/CMakeLists.txt [hunk 2]
```diff
@@ -77,9 +100,15 @@ if(INSTALL_HEADERS)
     if(WIN32)
         set(HEADER_EXCLUDE_REGEX "/apple|/ios|/linux|/mac|/solaris|/android|/dwarf|/tests|/testdata|/unittests")
     elseif(APPLE)
-        set(HEADER_EXCLUDE_REGEX "/apple|/ios|/linux|/windows|/solaris|/android|/dwarf|/tests|/testdata|/unittests|/*proj|/gcov")
+        set(HEADER_EXCLUDE_REGEX "/apple|/ios|/linux|/windows|/solaris|/android|/dwarf|/tests|/testdata|/unittests|/sender|/testapp|/*proj|/gcov")
     else()
         set(HEADER_EXCLUDE_REGEX "/apple|/ios|/client/linux/data|/client/linux/sender|/windows|/mac|/solaris|/android|/dwarf|/tests|/testdata|/unittests")
+        install(
+            DIRECTORY src/third_party/lss
```
Comment (BillyONeal) [inline ports/breakpad/CMakeLists.txt:107]: This is a bit... suspect. Is lss an embedded/vendored third party library? Why are customers expected to `#include <third_party/anything>` ?

Comment (Perlmint) [reply ports/breakpad/CMakeLists.txt:107]: They are not included manually. But, Headers - should be included to use breakpad_client - include they as `third_party/*`.

Below is an error message from my test project without installing headers in `third_party/lss`.

```
In file included from /ws/installed/x64-linux/include/client/linux/dump_writer_common/thread_info.h:37,
                from /ws/installed/x64-linux/include/client/linux/minidump_writer/linux_dumper.h:54,
                from /ws/installed/x64-linux/include/client/linux/minidump_writer/minidump_writer.h:42,
                from /ws/installed/x64-linux/include/client/linux/handler/exception_handler.h:42,
                from /ws/test_proj/MinidumpHandler.cpp:12:
/ws/installed/x64-linux/include/common/memory_allocator.h:50:10: fatal error: third_party/lss/linux_syscall_support.h: No such file or directory
   50 | #include "third_party/lss/linux_syscall_support.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment (BillyONeal) [reply ports/breakpad/CMakeLists.txt:107]: Hmmm sounds OK.

**Comment by BillyONeal** [comment]
Thanks for your contribution!

**Pull request merged**